### ISSUE TITLE
Quote table names in generated SQL

### ISSUE THREAD

**Issue by bsutherland** [jenseng/hair_trigger#41]
We have a table named "user". In Postgres this is a reserved word, so the following SQL generated by hair_trigger fails:

`DROP TRIGGER IF EXISTS user_after_update_of_login_id_row_tr ON user;`

This could be solved by escaping table names in all generated SQL (in the case of postgres, using double quotes):

`DROP TRIGGER IF EXISTS user_after_update_of_login_id_row_tr ON "user";`

**Comment by jenseng** [comment]
Cool, good catch. I'll try to get a fix up for this soon. The quoting part is easy, just need to make sure we match regenerated trigger definitions (now with quotes) to existing ones in the db (possibly without), so we can avoid mismatches ... be nice to have the actual `create_trigger` calls in schema.rb, instead of adapter-specific `execute('CREATE TRIGGER..')`

**Comment by bsutherland** [comment]
Great, thanks. In the mean time I worked around it by using a manual migration definition and quoting the table name like this:

```ruby
create_trigger("user_after_insert_row_tr", compatibility: 1).
  on('"user"').
...
```

**Comment by jenseng** [comment]
@bsutherland note that once I push a gem and up upgrade to it, you'll probably want to go back and remove the quoting from your manual migration

**Comment by bsutherland** [comment]
@jenseng Thanks for the fixes!

**Comment by jenseng** [comment]
no problem, thanks for the bug reports :+1:

**Comment by bsutherland** [comment]
@jenseng, I've upgraded to the latest version, and went back to remove the quoting, but I'm having trouble.

This was what I had (to workaround the issue)

```
create_trigger("user_after_update_of_login_id_row_tr", compatibility: 1).
      on('"user"').
 ...
```
With the new version, this gets double-quoted and fails (as expected).

But, when I change to:

```
on('user').
```

It doesn't get quoted at all! I took a look at the commit that fixed it, but I'm not really sure what's going on. Your help would be greatly appreciated!
